changelogs/fragments/5540-adding-features-to-ilo_redfish_info.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - added new features for ilo_redfish_info  (https://github.com/ansible-collections/community.general/pull/5540).
```

More details should be included in the message.
```suggestion
  - ilo_redfish_info - add new features  (https://github.com/ansible-collections/community.general/pull/5540).
```